src/ipaddr.coffee

```diff
@@ -387,6 +387,15 @@ ipaddr.parseCIDR = (string) ->
     catch e
       throw new Error "ipaddr: the address has neither IPv6 nor IPv4 CIDR format"
 
+ipaddr.parseBinary = (bytes) ->
+  length = bytes.length
+  if length == 4
+    return new ipaddr.IPv4(bytes)
+  else if length == 8
+    return new ipaddr.IPv6(bytes)
```

Comment:
I know, but your parser (ipaddr.IPv6) expects an array of 8 16bit pieces. My second PR based on this changes that to support both 8x16bits and 16x8bits.